scripts/stage_downstream_rebase.sh

```diff
@@ -61,13 +61,13 @@ sed -i 's/deploymentName: webhook/deploymentName: osp-director-operator-controll
 # Convert any tags to digests within the CSV (for offline/air gapped environments)
 for csv_image in $(cat bundle/manifests/osp-director-operator.clusterserviceversion.yaml | grep "image:" | sed -e "s|.*image:||" | sort -u); do
   base_image=$(echo $csv_image | cut -f 1 -d':')
-  if [[ "$base_image" == "registry.redhat.io/rhosp-rhel8-tech-preview/osp-director-operator" ]]; then
-    sed -i "s|$base_image:$tag_image|OPERATOR_IMG_WITH_DIGEST|g" bundle/manifests/osp-director-operator.clusterserviceversion.yaml
+  tag_image=$(echo $csv_image | cut -f 2 -d':')
+  if [[ "$base_image:$tag_image" == "controller:latest" ]]; then
+    sed -e "s|$base_image:$tag_image|$IMG|g" -i bundle/manifests/osp-director-operator.clusterserviceversion.yaml
   else
-    tag_image=$(echo $csv_image | cut -f 2 -d':')
     digest_image=$(skopeo inspect docker://$base_image:$tag_image | jq '.Digest' -r)
```

Comment:
might be worth checking for an empty digest, happens when skopeo fails e.g with auth failure

Reply:
@olliewalsh added. Thanks!